src/inmanta/app.py

```diff
@@ -964,7 +964,17 @@ def default_logging_config(options: argparse.Namespace) -> None:
     context = {var: f"{place_holder}{var}{place_holder}" for var in context_vars}
 
     component_config = InmantaLoggerConfig(stream=sys.stdout, no_install=True)
-    component_config.apply_options(options, options.cmd, context)
+    # Force TTY so that this command outputs the same config when piping to a file
+    original_force_tty = os.environ.get(const.ENVIRON_FORCE_TTY, None)
+    os.environ[const.ENVIRON_FORCE_TTY] = original_force_tty if original_force_tty else "yes"
+    try:
+        component_config.apply_options(options, options.cmd, context)
+    except Exception as e:
+        raise e
+    finally:
+        # Revert this env var back to its original state
+        if original_force_tty is None:
+            del os.environ[const.ENVIRON_FORCE_TTY]
```

Comment:
* The except clause is redundant.
* If `original_force_tty` is set to an empty string, we will overwrite the value with "yes". This makes the comment "Revert this env var back to its original state" invalid.

```suggestion
    # Force TTY so that this command outputs the same config when piping to a file
    original_force_tty = os.environ.get(const.ENVIRON_FORCE_TTY, None)
    if original_force_tty is None:
        os.environ[const.ENVIRON_FORCE_TTY] = "yes"
    try:
        component_config.apply_options(options, options.cmd, context)
    finally:
        # Revert this env var back to its original state
        if original_force_tty is None:
            del os.environ[const.ENVIRON_FORCE_TTY]
```